src/util/support/plugins.c

```diff
@@ -64,10 +64,12 @@
  */
 #ifdef __linux__
 #include <features.h>
+#if defined(__GLIBC__)
```

Comment:
Is there a reason not to use `#ifdef` here?

Reply:
No, I will change it.

Reply:
Thanks for the git commit link.